packages/server/assets/workspacesCore/typedefs/workspaces.graphql

```diff
@@ -1,3 +1,72 @@
+extend type Query {
+  workspace(id: String!): Workspace!
+}
+
+input WorkspaceCreateInput {
+  name: String!
+  description: String
+  logoUrl: String
+}
+
+input WorkspaceUpdateInput {
+  id: String!
+  name: String
+  description: String
+  logoUrl: String
+}
+
+input WorkspaceRoleUpdateInput {
+  userId: String!
+  workspaceId: String!
+  role: WorkspaceRole!
+}
+
+input WorkspaceRoleDeleteInput {
+  userId: String!
+  workspaceId: String!
+}
+
+extend type Mutation {
+  workspaceMutations: WorkspaceMutations!
```

Comment:
@hasScore might be necessary? @hasServerRole for sure is necessary, you should be authed to do any of this